bpf/tests/tc_nodeport_lb4_dsr_backend.c

```diff
@@ -13,6 +13,7 @@
 #define ENABLE_IPV4
 #define ENABLE_NODEPORT
 #define ENABLE_DSR		1
+#define REMOVE_DSR_IP_OPTION 1
```

Comment:
indentation could be made consistent here as well

Reply:
Thanks, done. LMK if this is what you had in mind.